README.md

```diff
@@ -31,53 +31,7 @@ _Lightweight_ &bull; _Modern_ &bull; _Accessibile_ &bull; _Customisable_ &bull;
 
 Our features are always checked against the W3C accessibility guidelines.
 
-A list of every a11y concern Checka11y.css will check for and highlight:
```

Comment:
Sorry if the issue description was too brief, I was thinking we could leave this section as it but add a note below this section saying, "For further details on the features listed above, please refer to the [feature references](./references.md)". It's really useful for a developer to come straight to this README and be able to get a really quick overview. 😊

Reply:
Ah sure, that's no problem - I probably should have clarified that with you but felt we may end up duplicating effort by keeping the README and references files updated with similar info